src/content/en/fundamentals/design-and-ui/ux-basics/index.md

```diff
@@ -0,0 +1,234 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: A step by step guide of the basics of UX design
+
+
+{# wf_updated_on: 2016-10-1 #}
+{# wf_published_on: 2016-10-1 #}
+
+# Basics of UX {: .page-title }
+
+{% include "web/_shared/contributors/mustafa.html" %}
+
+This article introduces a workflow that can help teams, products, startups and companies create a robust and meaningful process for developing a better user experience for their customers. You could use different parts of the process separately but they ideally work best as a series of steps. This guide borrows heavily from the Design Sprint methodology that multiple teams across Google have used to troubleshoot and solve challenges such as the [Self Driving Car](https://www.google.com/selfdrivingcar/ "Self Driving Car"){:target="_blank" .external}  and [Project Loon](https://www.solveforx.com/loon/ "Project Loon"){:target="_blank" .external}.
+
+### Double Diamond
+
+This flow work is based on what we in UX circles call the double diamond, made popular by the [British Design Council](http://webarchive.nationalarchives.gov.uk/20080821115409/designcouncil.org.uk/en/about-design/managingdesign/the-study-of-the-design-process/ "British Design Council"){:target="_blank" .external}, where your team diverges to understand an idea through research and then converges to define on the challenge, diverges to sketch it individually,  share the ideas, decide on what the best way forward, test and validate.
+
+<figure>
+  <img src="images/double-diamond.png" alt="Phases of a project include; Understand, Define, Diverge, Decide, Prototype and Validate">
```

Comment:
You've included 1x and 2x images in the PR, but are only serving the 1x versions. You can either drop the 2x version, or serve the 2x and DevSite will resize images for you.